src/Kunstmaan/MediaBundle/Tests/unit/Helper/Image/ImageHandlerTest.php

```diff
@@ -0,0 +1,33 @@
+<?php
+
+namespace Kunstmaan\MediaBundle\Tests\Helper\Image;
+
+use Kunstmaan\MediaBundle\Entity\Media;
+use Kunstmaan\MediaBundle\Helper\Image\ImageHandler;
+use Kunstmaan\UtilitiesBundle\Helper\Slugifier;
+use PHPUnit\Framework\TestCase;
+use Symfony\Component\HttpFoundation\File\File;
+use Symfony\Component\HttpFoundation\File\MimeType\MimeTypeGuesser;
+use Kunstmaan\MediaBundle\Helper\MimeTypeGuesserFactoryInterface;
+use Kunstmaan\MediaBundle\Helper\ExtensionGuesserFactoryInterface;
+
+class ImageHandlerTest extends TestCase
+{
+    public function testPrepareWithSvg()
+    {
+        $mockMimeTypeGuesserfactory = $this->createMock(MimeTypeGuesserFactoryInterface::class);
+        $mockExtensionGuesserfactory = $this->createMock(ExtensionGuesserFactoryInterface::class);
+
+        $mockMimeTypeGuesserfactory->method('get')->willReturn(MimeTypeGuesser::getInstance());
+
+        $handler = new ImageHandler(1, $mockMimeTypeGuesserfactory, $mockExtensionGuesserfactory, 'aviaryKey');
+        $handler->setSlugifier(new Slugifier());
+
+        $media = new Media();
+        $media->setContent(new File(__DIR__.'/../../Fixtures/sample.svg'));
+
+        $handler->prepareMedia($media);
+
+        $this->assertSame(['original_width' => null, 'original_height' => null], $media->getMetadata());
```

Comment:
```suggestion
        self::assertSame(
            ['original_width' => null, 'original_height' => null],
            $media->getMetadata()
        );
```

Reply:
I'm going to leave this for now, as we use the `$this->` "syntax" everywhere. So we can decided later what to use and php-cs-fixer to switch all usages at once.